changelogs/fragments/259-add-routeros7-support-for-ip-vrf.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - api_data - add support for the ``ip vrf`` path in RouterOS 7 and add support new primary_keys ``routing-mark`` for RouterOS 7 (https://github.com/ansible-collections/community.routeros/pull/259)
+  - api_modify, api_info - add support for the ``ip vrf`` path in RouterOS 7  (https://github.com/ansible-collections/community.routeros/pull/259)
```

Comment:
```suggestion
  - api_modify, api_info - add support for the ``ip vrf`` path in RouterOS 7 and support new primary keys ``routing-mark`` for RouterOS 7 (https://github.com/ansible-collections/community.routeros/pull/259)
```

Reply:
I copied that part over from the other part that was removed. If it's wrong, I don't mind not having this here :)